setup.py

```diff
@@ -44,14 +44,12 @@
     license="BSD",
     zip_safe=False,
     keywords='dateparser',
-    python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*',
+    python_requires='!=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*',
     classifiers=[
         'Development Status :: 2 - Pre-Alpha',
```

Comment:
I have just realized that we are in "Pre-Alpha" status.... :thinking:  
We should probably update this to be `Development Status :: 4 - Beta` or  `Development Status :: 5 - Production/Stable` @Gallaecio let me know your thoughts.

Reply:
I'm not totally confident in moving to "Production/Stable" because of the "multiple thread" issues: https://github.com/scrapinghub/dateparser/issues/441

Reply:
I’m not sure whether or not multithreading support is a requirement for Dateparser to be considered stable, but I think it makes sense to delay that tag until Dateparser 1.0 is released.

Reply:
@Gallaecio  hmmm.. but these changes are performed to release that 1.0.0 version, so in that case, it makes sense to change it now. right?

Reply:
Oh, I forgot 1.0 was the next version! In that case, I would go for that tag, yes. Stable does not mean feature-complete. It’s production-ready for people that do not need multiprocessing support.